.travis.yml

```diff
@@ -5,14 +5,14 @@ env:
 - API_REGEX='Google\.LongRunning'
 language: csharp
 mono: none
-dotnet: 2.1.400
-dist: trusty
+dotnet: 2.2.104
+dist: xenial
 
 # Install the .NET Core 1.0 runtime as that's what we test against
 addons:
   apt:
     sources:
-    - sourceline: 'deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-ubuntu-trusty-prod trusty main'
+    - sourceline: 'deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-ubuntu-xenial-prod trusty main'
```

Comment:
question: there's still a "trusty" at the end of the new line, before main, not sure if that is correct.

Reply:
I'm surprised that even works! Will edit and see if it still does afterwards...